README.md

```diff
@@ -213,15 +213,16 @@ The `raw commitment` is an RLP-encoded [EigenDA certificate](https://github.com/
 Unit tests can be ran via invoking `make test`.
 
 ### Integration
-End-to-end (E2E) tests can be ran via `make e2e-test`.
 
-### Holesky
+End-to-end (E2E) tests can be ran via `make e2e-test`. These tests use the [op-e2e](https://github.com/ethereum-optimism/optimism/tree/develop/op-e2e) framework for asserting correct interaction behaviors with batch submission and state derivation.
```

Comment:
Why did we switch from calling the make target `optimism-test` to `e2e-test`? I'm guessing these e2e test also test some nitro stuff? If not, should we just change the name back to `test-optimism-integration`? cc @epociask

Reply:
we initially had a separate entry-point (i.e, OP env flag) for these tests independent of the other E2E tests. Now `e2e-test` is a superset of `optimism-test`

Reply:
> test some nitro stuff 

Nop we currently don't do any E2E nitro tests in proxy directly. Could be worth exploring in the future

Reply:
let's also add an additional section specifying that these tests also assert E2E client <-> server interactions using simple/op clients

Reply:
Added verbatim [c5d6f0d](https://github.com/Layr-Labs/eigenda-proxy/pull/225/commits/c5d6f0d4f4fed70d9debba5355d468af22e98613)